test/test_ops.py

```diff
@@ -385,6 +460,88 @@ def test_vmapvjp(self, device, dtype, op):
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(fn, args, {}):
                 self.assertEqual(loop_out, batched_out, atol=1e-4, rtol=1e-4)
 
+    # There are several variations we care about
+    # 1) primal batched (TODO)
+    # 2) tangent batched (batched grads) <--
+    # 3) both batched (TODO)
+    # The below tests (2) only.
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_vmapjvp', {
+        xfail('nn.functional.dropout'),  # randomness
+
+        # TODO: fails in core due to in-place batched nto non-batched
+        # but fails here for a different reason
+        xfail('linalg.householder_product'),
+
+        # Try to in-place batched tensor into non-batched tensor
+        xfail('matrix_exp'),
+        xfail('lu'),
+        xfail('fill_'),
+        xfail('block_diag'),  # TODO: We expect this to fail in core, but it doesn't
+        xfail('index_add'),
+        xfail('index_copy'),
+        xfail('index_put'),
+        xfail('index_fill'),
+        xfail('masked_fill'),
+        xfail('masked_scatter'),
+
+        # =============================================
+        # NB: The above failures also fail in core.
+        #     The failures below only fail in functorch
+        # =============================================
+
+        # Expected the output of forward differentiable view operations
+        # to have the tangent have the same layout as primal
+        xfail('reshape'),  # this is odd
+        xfail('linalg.tensorinv'),
+
+        # Composite ops that do bad things
+        # RuntimeError: Cannot access data pointer of Tensor that doesn't have storage
+        xfail('tensor_split'),
+        xfail('linalg.inv'),
+        xfail('linalg.matrix_power'),
+        xfail('linalg.cholesky'),
+
+        # Apprently these support forward AD, but we get "Trying to use forward AD..."
+        # These are cases where OpInfo has supports_forward_ad=True, but disables the test
+        xfail('var_mean'),
+        xfail('std_mean'),
+        xfail('linalg.eigvalsh'),
+
+        # These are issues that should be fixed in core. See repro in core:
+        # https://github.com/pytorch/functorch/pull/232#discussion_r751405155
+        # RuntimeError: expected scalar type double but found float
+        xfail('minimum'),
+        xfail('min', 'binary'),
+        xfail('maximum'),
+        xfail('max', 'binary'),
```

Comment:
Let's move these above the "The above failures also fail in core." line. The thing we're interested in testing is if a user uses PyTorch core and encounters the problem, not if the PyTorch test suite says there is a problem. These two things would ideally be the same but they aren't because the PyTorch test suite currently only runs jvp tests in double precision

Reply:
Good point.